### PR TITLE
build!: drop IE11 support

### PR DESCRIPTION
**Pull request by kris-ellery**
Removed IE11 transforms from styleguide config.

### CODE REVIEW

File: lib/configs/createStyleguideConfig.js [hunk 1]
```diff
@@ -189,42 +189,6 @@ module.exports = (config = {}, options = {}) => {
...
-            'estree-walker',
-            'regexpu-core',
-            'unicode-match-property-ecmascript',
-            'unicode-match-property-value-ecmascript',
-            'react-dev-utils',
-            'ansi-styles',
-            'ansi-regex',
-            'chalk',
-            'strip-ansi',
-            ...(options.ie11ModuleTransforms || []),
```
Comment (stonebk) [inline lib/configs/createStyleguideConfig.js:205]: Do you want to delete the documentation about this `ie11ModuleTransforms` option?

Comment (stonebk) [reply lib/configs/createStyleguideConfig.js:205]: https://github.com/zillow/create-react-styleguide#createstyleguideconfigconfig-options

Comment (kris-ellery) [reply lib/configs/createStyleguideConfig.js:205]: Great catch! Removed.

Comment (stonebk) [reply lib/configs/createStyleguideConfig.js:205]: You might also want to update your commit with notes about dropping that option for the CHANGELOG.

**Review by stonebk — APPROVED** [review]

**Pull request merged**